cmd/aggregated-apiserver/app/aggregated-apiserver.go

```diff
@@ -18,8 +18,10 @@ func NewAggregatedApiserverCommand(ctx context.Context) *cobra.Command {
 	opts := options.NewOptions()
 
 	cmd := &cobra.Command{
-		Use:  "karmada-aggregated-apiserver",
-		Long: `Launch the karmada-aggregated-apiserver`,
+		Use: "karmada-aggregated-apiserver",
+		Long: `The karmada-aggregated-apiserver starts an aggregated server. 
+It is responsible for managing cluster objects and provides the ability to aggregate APIs, 
+allowing users to access member clusters in the control plane.`,
```

Comment:
It could be not clear. How is it below?

```suggestion
allowing users to access member clusters from the control plane directly.`,
```

Reply:
Done.